core/src/avm2/globals/flash/events/UncaughtErrorEvent.as

```diff
@@ -0,0 +1,36 @@
+package flash.events
+{
+    
+    public class UncaughtErrorEvent extends ErrorEvent
+    {
+        public static const UNCAUGHT_ERROR:String = "uncaughtError"; // [static] Defines the value of the type property of an uncaughtError event object.
+
+        public var _error: *; // [read-only] The error object associated with the uncaught error.
+
+        public function UncaughtErrorEvent(type:String, bubbles:Boolean = true, cancelable:Boolean = true, error_in:* = null)
+        {
+            super(type,bubbles,cancelable);
+            this._error = error_in;
+        }
+        
+
+        // [override] Creates a copy of the UncaughtErrorEvent object and sets the value of each property to match that of the original.
+        override public function clone():Event
+        {
+            return new UncaughtErrorEvent(this.type, this.bubbles, this.cancelable, this.text, this.errorID, );
```

Comment:
As the CI checks pointed out, there is a syntax error here, at the end of the line.